src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx

```diff
@@ -101,24 +94,7 @@ function AppointmentsPage({
         )}
       />
 
-      {showScheduleButton && (
-        <>
-          {!featureCovid19Vaccine && (
-            <ScheduleNewAppointment
```

Comment:
This old component was mainly just choosing between different pieces of text by feature flag. It doesn't seem useful to keep around, since the current CTA works regardless of what feature flags are on.